README.md

```diff
@@ -1 +1,8 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+
+This project is about building a Portfolio using the mobile-first approach. This is the first step of the whole project for my portfolio. According to project requirements, we must follow one of the designs from provided templates to design the website and I have decided to follow the "Template 2" design and look from Figma.
+
+
+## Build with
+
+- HTML & CSS
```

Comment:
- From the project requirement you should add a descriptive readme.md file to your project. But your readme file only has the `project name` and `built with` sections. Please kindly add the rest of the sections to your project readme.md file. You can follow Microverse readme [template](https://github.com/microverseinc/readme-template) to add the rest of the sections to your project readme.md file. :pray: